setup.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+### Shell script to spin up a docker container for oracle.
+
+## color codes
+RED='\033[1;31m'
+GREEN='\033[1;32m'
+YELLOW='\033[1;33m'
+CYAN='\033[1;36m'
+PLAIN='\033[0m'
+
+## variables
+ORACLE_CONTAINER="oracle_c"
+HOST="localhost"
+PORT=1521
+DATABASE="XE"
+USER="system"
+PASSWORD="oracle"
+if [ "$1" ]; then
+    HOST=$1
+fi
+if [ "$2" ]; then
+    PORT=$2
+fi
+
+
+## check if docker exists
+printf "\n${RED}>> Checking for docker${PLAIN} ${GREEN}...${PLAIN}"
+docker -v > /dev/null 2>&1
+DOCKER_EXISTS=$?
+if [ "$DOCKER_EXISTS" -ne 0 ]; then
+    printf "\n\n${CYAN}Status: ${PLAIN}${RED}Docker not found. Terminating setup.${PLAIN}\n\n"
+    exit 1
+fi
+printf "\n${CYAN}Found docker. Moving on with the setup.${PLAIN}\n"
+
+## cleaning up previous builds
+printf "\n${RED}>> Finding old builds and cleaning up${PLAIN} ${GREEN}...${PLAIN}"
+docker rm -f $ORACLE_CONTAINER > /dev/null 2>&1
+printf "\n${CYAN}Clean up complete.${PLAIN}\n"
+
+## pull latest oracle image
+printf "\n${RED}>> Pulling latest oracle image${PLAIN} ${GREEN}...${PLAIN}"
+docker pull sath89/oracle-xe-11g:latest > /dev/null 2>&1
+printf "\n${CYAN}Image successfully built.${PLAIN}\n"
+
+## run the oracle container
+printf "\n${RED}>> Starting the oracle container${PLAIN} ${GREEN}...${PLAIN}"
+docker run --name $ORACLE_CONTAINER -p $PORT:1521 -d sath89/oracle-xe-11g:latest 2>&1
+
+##wait for orale database container to be ready
+OUTPUT=1
+TIMEOUT=300
+TIME_PASSED=0
+WAIT_STRING="+"
+START_MESSAGE="Database ready to use."
+printf "${RED}Waiting for database to be ready${PLAIN} ${GREEN}...${PLAIN}"
```

Comment:
Add a new line at the front and end of this:
`printf "\n${RED}Waiting for database to be ready${PLAIN} ${GREEN}...${PLAIN}\n"`